src/build/build_helpers.jl

```diff
@@ -142,12 +142,18 @@ function get_expression_for_commodity_producer_price(jm, pf, commodity::Commodit
         for output in pf.outputs
             if output.commodity == commodity
                 for tax in output.taxes
-                    push!(taxes, tax.rate)
+                    push!(taxes, eval(swap_our_param_with_val(tax.rate)))
```

Comment:
```suggestion
                    push!(taxes, tojump(jm, tax.rate))
```

What we want to do here is convert the Julia expression in `tax.rate` to a JuMP expression, not eval the thing.